CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ NDM uses the standard GitHub pull requests process to review and accept contribu
 * If you want to file an issue for a bug or feature request, please see [Filing a issue](#filing-an-issue)
 * If you have documentation improvement ideas, go ahead and create a pull request. See [Pull Request checklist](#pull-request-checklist)
 * If you would like to make code contributions, please start with [Setting up the Development Environment](#setting-up-your-development-environment).
-* If you would like to work on something more involved, please connect with the OpenEBS Contributors. See [OpenEBS Community](https://github.com/openebs/openebs/tree/master/community)
+* If you would like to work on something more involved, please connect with the OpenEBS Contributors. See [OpenEBS Community](https://github.com/openebs/openebs/tree/main/community)
```

Comment:
should this link have `/tree/main/` in it? Shouldn't it be `/tree/develop/`?

Reply:
For repositories [openebs/openebs](https://github.com/openebs/openebs) and [linux-utils](https://github.com/openebs/linux-utils), `main` will be the default branch. for other repos `develop` will be used as release branches are created and releases are done from release branch. cc: @kmova

Reply:
Also, the branch creation/renaming will be done after the PR gets merged,